components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -954,6 +990,30 @@ - (void)fhv_accumulatorDidChange {
 
 #pragma mark Layout
 
+- (CGFloat)fhv_accumulatorMin {
+  CGFloat offsetWithoutInset = [self fhv_contentOffsetWithoutInjectedTopInset];
+  CGFloat headerHeight = -offsetWithoutInset;
```

Comment:
Is `offsetWithoutInset` being used? Why not just do `CGFloat headerHeight = -[self fhv_contentOffsetWithoutInjectedTopInset];`